api/src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -207,6 +226,9 @@ class GetVenueResponseModel(base.BaseVenueResponse, AccessibilityComplianceMixin
     collectivePhone: str | None
     collectiveEmail: str | None
     collectiveSubCategoryId: str | None
+    collectiveDmsApplications: list[DMSApplicationForEAC]
+    hasAdageId: bool = bool(offerers_models.Venue.adageId)
```

Comment:
attention si tu met ça il retournera toujours True.

Il faut le setter dans la méthode `from_orm`
```suggestion
    hasAdageId: bool
```